tephra-core/src/main/java/org/apache/tephra/distributed/thrift/TBoolean.java

```diff
@@ -24,19 +24,29 @@
  */
 package org.apache.tephra.distributed.thrift;
 
-import org.apache.thrift.EncodingUtils;
```

Comment:
Note that thrift does not generate imports in the order that the checkstyle rules expect. The previous version of the generated thrift stubs was apparently manually changed to reflect the correct order and to have a copyright. For the copyright, I added an instruction how to add that to the README file, but for import order, we really should not fix that manually every time we run thrift. So we see a lot of diffs in the import section here.